test/unit_test/deepdoc/parser/test_paddleocr_parser.py

```
@@ -0,0 +1,381 @@
+    assert post.call_args.kwargs["headers"]["Authorization"] == "Bearer tok"
+
+
+@pytest.mark.p1
+def test_self_hosted_omits_authorization_over_plain_http(monkeypatch):
+    _clear_env(monkeypatch)
+    module = _load_paddleocr_parser(monkeypatch)
+
+    post = Mock(return_value=_local_response())
+    monkeypatch.setattr(module.requests, "post", post)
+
+    parser = module.PaddleOCRParser(base_url="http://svc", access_token="tok")
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "access_token"

(S106)